Release 1.0.0 changes (#226)
eclipse/che-che4z-explorer-for-endevor: CHANGELOG.md

You can find all notable changes to Explorer for Endevor in this document.

## [1.0.0] &ndash; 2022-02-23

### Added

> - Telemetry event recording.

Reviewer: ```suggestion
- Added the telemetry event-recording functionality.
```

> - Added a `[MAP]` folder in the tree view. It is being populated with elements found up the map.

Reviewer: ```suggestion
- Added a `[MAP]` folder to the tree view. You can use the folder to store the elements that are found up the map.
```

> - Added a button `Show logs` in the notification panel which allows the user to open the extension's output.

Reviewer: ```suggestion
- Added a `Show logs` button to the notification panel, which enables you to open the output of the extension.
```

### Changed

- Multiple error logging and notification events.

### Fixed

> - Fixed an issue in the URI building process. Characters that may potentially break the building process are now being escaped.

Reviewer: ```suggestion
- Fixed an issue in the URI building process. Characters that may potentially break the building process are escaped.
```

> - Upload dialogs now prevent wildcards as input.

Reviewer: Is that a bug fix or a change?

Author: I would vote for a bugfix.

@nickImbirev what do you think?

Reviewer: If it's a bug fix, then it should read:

```suggestion
- Fixed an issue that erroneously allowed upload dialogs to use wildcards as input.
```

If not a bug fix, then discard the suggestion

> - Fixed an issue with conflict resolution causing the command to fail when resolving reoccurring conflicts.

Reviewer: ```suggestion
- Fixed an issue that caused the conflict resolution command to fail when resolving reoccurring conflicts.
```

> - Fixed an issue with retrieving, editing multiple elements with auto signout option. Now it processes all the selected elements in groups by each location profile.

Reviewer: ```suggestion
- Fixed an issue with retrieving and editing multiple elements with the auto signout option. Now auto signout processes all the selected elements in groups by location profile.
```

Reviewer: @roman-kupriyanov That sounds good to me. Great suggestion

Author: ```suggestion
- Fixed an issue with retrieving and editing multiple elements with the auto signout option. Now the retrieve and edit commands correctly process all the selected elements in groups by location profiles.
```

- Fixed an issue with showing multiple elements for editing. An "editor already opened" error does not appear anymore.

> - Incorrect editors closure when discarding changes.

Reviewer: ```suggestion
- Fixed an issue that caused editors to close when discarding changes.
```
